index.html

```diff
@@ -19,6 +20,34 @@
   <header>
   </header>
   <main>
+    <div data-view="entry-form">
+      <div class="head-background">
+        <div class="container">
+          <p class="journal-head">Code Journal</p>
```

Comment:
You should put this into a proper `row` and `column` to follow proper layout class setup.

Reply:
I made a change to have them in a row and column instead but I had to keep the background outside of the row as the `column-full` has a `padding: 0.75rem` when the screen has a `min-width: 768px.` Because of this change I was able to remove some extra lines of CSS that the `row` and `column` layout set.
![image](https://user-images.githubusercontent.com/109567499/189771913-8d1bb154-cd3a-4db0-8cd4-665ccfdcf84c.png)